workflows/sarek.nf

```diff
@@ -83,14 +83,9 @@ for (param in checkPathParamList) if (param) file(param, checkIfExists: true)
 */
 
 // Initialize file channels based on params, defined in the params.genomes[params.genome] scope
-ascat_alleles           = params.ascat_alleles           ? Channel.fromPath(params.ascat_alleles).collect()           : Channel.empty()
-ascat_loci              = params.ascat_loci              ? Channel.fromPath(params.ascat_loci).collect()              : Channel.empty()
-ascat_loci_gc           = params.ascat_loci_gc           ? Channel.fromPath(params.ascat_loci_gc).collect()           : Channel.value([])
-ascat_loci_rt           = params.ascat_loci_rt           ? Channel.fromPath(params.ascat_loci_rt).collect()           : Channel.value([])
```

Comment:
I see what you are doing. I would love if we can find a solution in the future where the file channel building happens in one place and not scattered.

Reply:
so do I